src/main/java/mate/academy/spring/boot/controller/BookController.java

```diff
@@ -17,38 +19,35 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
+@RequiredArgsConstructor
 @RestController
 @RequestMapping("/books")
 public class BookController {
     private final BookService bookService;
 
-    @Autowired
-    public BookController(BookService bookService) {
-        this.bookService = bookService;
-    }
-
-    @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
-    }
```

Comment:
you still need endpoint to get all Books

Reply:
please pay attention to the previous mentor's comment